backend/analytics_server/mhq/api/resources/settings_resource.py

```diff
@@ -55,6 +56,15 @@ def _add_setting_data(config_settings: ConfigurationSettings, response):
                 "default_sync_days": config_settings.specific_settings.default_sync_days
             }
 
+        if isinstance(config_settings.specific_settings, IncidentPrsSetting):
+            response["setting"] = {
+                "include_revert_prs": config_settings.specific_settings.include_revert_prs,
+                "title_filters": config_settings.specific_settings.title_filters,
+                "head_branch_filters": config_settings.specific_settings.head_branch_filters,
+                "pr_mapping_field": config_settings.specific_settings.pr_mapping_field,
+                "pr_mapping_pattern": config_settings.specific_settings.pr_mapping_pattern,
```

Comment:
We can have these settings in the subsequent prs since they are not related to cfr for now. This setting can be extended later on